### PR TITLE
Testing someone else submitting a PR

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -25,4 +25,13 @@
 .gross {
 	mask: url('...');
 	-webkit-appearance: textfield;
+}
+
+.test {
+	-webkit-appearance: textfield;
+	-moz-appearance: blue;
+	appearance: blah;
```
Comment (darkwing) [inline main.css:33]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

File: main.css [hunk 2]
```diff
@@ -25,4 +25,13 @@
 .gross {
 	mask: url('...');
 	-webkit-appearance: textfield;
+}
+
+.test {
+	-webkit-appearance: textfield;
+	-moz-appearance: blue;
```
Comment (darkwing) [inline main.css:32]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

File: main.css [hunk 3]
```diff
@@ -25,4 +25,13 @@
...
+	appearance: blah;
+
+	mask: url('boo');
```
Comment (darkwing) [inline main.css:35]: CSS Masks not supported by: IE (10,11), Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3), IE Mobile (10,11)

File: main.css [hunk 4]
```diff
@@ -25,4 +25,13 @@
 .gross {
 	mask: url('...');
 	-webkit-appearance: textfield;
+}
+
+.test {
+	-webkit-appearance: textfield;
```
Comment (darkwing) [inline main.css:31]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

**Comment by openjck** [comment]
Woo! It worked.

File: main.css (hunk 4 above)
Comment (darkwing) [inline main.css:31]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

File: main.css (hunk 2 above)
Comment (darkwing) [inline main.css:32]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

File: main.css (hunk 1 above)
Comment (darkwing) [inline main.css:33]: CSS Appearance not supported by: IE (10,11), IE Mobile (10,11)

File: main.css (hunk 3 above)
Comment (darkwing) [inline main.css:35]: CSS Masks not supported by: IE (10,11), Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3), IE Mobile (10,11)

**Pull request closed without merging**